app/controllers/admin/publishers_controller.rb

```diff
@@ -22,6 +22,7 @@ def index
     if params[:two_factor_authentication_removal].present?
       @publishers = @publishers.joins(:two_factor_authentication_removal).distinct
     end
+    @publishers = @publishers.where("email IS NOT NULL or pending_email IS NOT NULL") # Don't include deleted users
```

Comment:
Let's try and use the ORM interface instead

```suggestion
    @publishers = @publishers.where.not(email: nil, pending_email: nil") # Don't include deleted users
```

Reply:
Your suggestion is an `AND` rather than `OR`. If you're able to figure out how to use this using the ORM interface, I welcome it, but the performance is still the same.

Reply:
I believe you can do `OR` with
```
@publishers.where.not(email: nil).or.@publishers.where.not(pending_email: nil)
```

Reply:
Let's just keep it the same, that way it's more readable